source/grid/grid_out.inst.in

```diff
@@ -24,6 +24,11 @@ for (deal_II_dimension : DIMENSIONS)
       const Triangulation<deal_II_dimension> &, std::ostream &) const;
 #endif
 
+#if deal_II_dimension != 2
```

Comment:
Why only if we're not in 2d? Where do we do the instantiation for the 2d case?

Reply:
The answer is in these lines:
https://github.com/dealii/dealii/blob/master/source/grid/grid_out.cc#L1520-L1531

- 2D is instantiated in the .cc-file
- for 1D/3D it is not working, but an exception should be thrown

Reply:
Ah, I see. Just remove the explicit instantiation of the 2d case in the .cc file, because they you won't need the special casing in the .inst.in file.

Reply:
Is the new commit like you described? I have replaced the explicit instantiation with an `Assert`. Is this fine, or should we add there an `AssertThrow`?